modules/models/record.php

```diff
@@ -123,6 +123,36 @@ public function delete($pk="id") // Deletes by assuming $id is always the pk. No
         return;
     }
 
+    public function deleteWhere($pk, $pk2) // Deletes from tables where pk is compound key
+    {
+        // Check to see if we are deleting a record that has not been saved yet.
+        if ($this->is_new_record) {
+            throw new ErrorException("Cannot delete an object that is not yet in a table. Please set the object to null instead.");
+        }
+
```

Comment:
i don't really like this function, I expected it to have a function signature similar to `where`, but im ok with overlooking this